apply-revapi-suggestions.java

```diff
@@ -0,0 +1,159 @@
+//usr/bin/env jbang "$0" "$@" ; exit $?
+
+import java.io.ByteArrayInputStream;
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.io.SequenceInputStream;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.io.File;
+import java.io.FileReader;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+/**
+ * This script applies the suggestions generated by the revapi output into the api-changes.xml file.
+ * The suggestions are in api-changes-suggestions.xml file. These suggestions are the new changes in the codebase that
+ * you might want to ignore (or otherwise process).
+ *
+ * The api-changes.xml file contains the new changes you want to ignore.
+ *
+ * This script record/update the suggestions that are in the api-changes-suggestions.xml into api-changes.xml.
+ */
+class jbangscript {
```

Comment:
name it the same as file - yes, its valid but many will get confused and not focus on the real value.